exercises/hamming/example.hs

```diff
@@ -1,4 +1,4 @@
-module DNA (hammingDistance) where
+module Hamming (hammingDistance) where
```

Comment:
Should this be just `distance` instead of `hammingDistance` to avoid the echo with the module name?